lib/elements/tab_expand_controller.dart

```diff
@@ -37,7 +37,7 @@ class TabExpandController {
   final List<StreamSubscription<Event>> _subscriptions = [];
 
   late TabState _state;
-  late Splitter _splitter;
+  Splitter? _splitter;
```

Comment:
This can still be `late` I think

Reply:
> This can still be `late` I think

Yes, I just tried and it does not seem to be throwing `LateInitializationError`.